src/pages/game/is-guessed-toast/isGuessedToastSlice.ts

```diff
@@ -0,0 +1,56 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { RootState } from '../../../app/store';
+import { isGuessedToastSliceType } from '../../../types/isGuessedToastSlice';
+import { POLISH_COMMON_WORDS } from '../../../utils/constants';
+import { setGuessText } from '../guess-bar/guessSlice';
+
+const initialState: isGuessedToastSliceType = {
+    guessedWords: [],
+    isAlreadyGuessed: false,
+};
+
+export const guessedWordsSlice = createSlice({
```

Comment:
instead of creating new slice I would extend guessSlice as it is almost empty and can share this context